ajax/libs/taggle/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "taggle",
+  "filename": "example/js/taggle.min.js",
+  "version": "1.7.1",
+  "description": "Taggle is a dependency-less tagging library",
+  "author": "Sean Coker <sean@seancoker.com>",
+  "url": "http://sean.is/poppin/tags",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:okcoker/taggle.js.git"
+  },
+  "homepage": "https://sean.is/poppin/tags",
+  "keywords": [
+    "tags",
+    "tagging",
+    "standalone",
+    "micro",
+    "library"
+  ],
+  "license": "MIT",
+  "npmName": "taggle",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "example/js/taggle*"
```

Comment:
Example is not a distribution purpose directory, it's wired to get files from an example, unless you've confirmed with author to make sure that the example will be updated in every release.